ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -53,6 +53,8 @@ final class Metrics(override val registry: MetricRegistry, val otelMeter: OtelMe
 
     object parallelIndexer extends ParallelIndexerMetrics(prefix :+ "parallel_indexer", registry)
 
+    object indexerEvents extends UpdateEventsMetrics(prefix :+ "indexer", openTelemetryFactory)
```

Comment:
Nit: These `object`s seem to be in random order, however, I'd suggest putting this new set right under `indexer` as it also has the `indexer` prefix.